docs/source/backends/zerotier.rst

```diff
@@ -315,18 +313,15 @@ Automatic generation of clients
 
 .. automethod:: netjsonconfig.OpenWrt.zerotier_auto_client
 
-Example:
+**Example (with custom ``zerotier interface name``)**:
```

Comment:
why is `zerotier interface name` highlighted as code here?

Reply:
Fixed, thanks!